### ISSUE TITLE
vmbackup error when specifying -origin and using Azure Storage Account

### ISSUE THREAD

**Issue by mckennajones** [VictoriaMetrics/VictoriaMetrics#3477]
**Describe the bug**
When using the vmbackup tool with an Azure Storage account as a storage medium, and specifying the `-origin` flag to point to an existing (incremental) backup to speed the process up via a server-side copy, the tool encounters a 409 error from Azure. (Logs below). 

It is unclear if this is a result of the way that the vmbackup tool interacts with the Azure Storage account, or if it could be a permissions/quota issue on my Azure account. 

**To Reproduce**
I have two K8s cron jobs setup, one performs incremental backups (by always pointing to the same folder in the storage account) daily, then there is a weekly job that points to a different path, and speicfies the `-origin` flag pointing to the incremental backup, at `/latest`. Example `vmbackup` command:

```
the_date=$(date +%F)
./vmbackup-prod \
-storageDataPath=/storage \
-snapshot.createURL=http://victoria-metrics-victoria-metrics-cluster-vmstorage-1.victoria-metrics-victoria-metrics-cluster-vmstorage.victoria-metrics:8482/snapshot/create \
-dst=azblob://victoria-metrics/full/${the_date}/victoria-metrics-victoria-metrics-cluster-vmstorage-1 \
-origin=azblob://victoria-metrics/latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1
```

Credentials are provided via the `AZURE_STORAGE_ACCOUNT_CONNECTION_STRING` env var. 

The backup at the `-origin` location is about 500GB.

**Expected behavior**
The backup should succeed using a server-side copy when `-origin` is specified. 

**Logs**
```
➜  ~ k8s:(mr-admin-A) k logs -n victoria-metrics vmstorage-1-full-manual-ldjmk -f
2022-12-12T19:43:51.912Z	info	VictoriaMetrics/lib/logger/flag.go:12	build version: vmbackup-20221007-005245-tags-v1.82.0-0-g832276064
2022-12-12T19:43:51.912Z	info	VictoriaMetrics/lib/logger/flag.go:13	command-line flags
2022-12-12T19:43:51.912Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -dst="azblob://victoria-metrics/full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1"
2022-12-12T19:43:51.912Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -origin="azblob://victoria-metrics/latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1"
2022-12-12T19:43:51.912Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -snapshot.createURL="http://victoria-metrics-victoria-metrics-cluster-vmstorage-1.victoria-metrics-victoria-metrics-cluster-vmstorage.victoria-metrics:8482/snapshot/create"
2022-12-12T19:43:51.912Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -storageDataPath="/storage"
2022-12-12T19:43:51.912Z	info	VictoriaMetrics/app/vmbackup/main.go:53	Snapshot create url http://victoria-metrics-victoria-metrics-cluster-vmstorage-1.victoria-metrics-victoria-metrics-cluster-vmstorage.victoria-metrics:8482/snapshot/create
2022-12-12T19:43:51.912Z	info	VictoriaMetrics/app/vmbackup/main.go:60	Snapshot delete url http://victoria-metrics-victoria-metrics-cluster-vmstorage-1.victoria-metrics-victoria-metrics-cluster-vmstorage.victoria-metrics:8482/snapshot/delete
2022-12-12T19:43:51.912Z	info	VictoriaMetrics/lib/snapshot/snapshot.go:28	Creating snapshot
2022-12-12T19:43:53.582Z	info	VictoriaMetrics/lib/snapshot/snapshot.go:52	Snapshot 20221212194351-172EA3346AA94EFA created
2022-12-12T19:43:53.583Z	info	VictoriaMetrics/lib/httpserver/httpserver.go:96	starting http server at http://127.0.0.1:8420/
2022-12-12T19:43:53.583Z	info	VictoriaMetrics/lib/httpserver/httpserver.go:97	pprof handlers are exposed at http://127.0.0.1:8420/debug/pprof/
2022-12-12T19:43:53.623Z	error	VictoriaMetrics/lib/backup/azremote/azblob.go:293	GetProperties("https://mradminmetricsbackups.blob.core.windows.net/victoria-metrics/full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/backup_complete.ignore") returned HEAD https://mradminmetricsbackups.blob.core.windows.net/victoria-metrics/full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/backup_complete.ignore
--------------------------------------------------------------------------------
RESPONSE 404: 404 The specified blob does not exist.
ERROR CODE: BlobNotFound
--------------------------------------------------------------------------------
Response contained no body
--------------------------------------------------------------------------------
2022-12-12T19:43:53.623Z	info	VictoriaMetrics/lib/backup/actions/backup.go:72	starting backup from fslocal "/storage/snapshots/20221212194351-172EA3346AA94EFA" to AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} using origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.633Z	info	VictoriaMetrics/lib/backup/actions/backup.go:78	obtained 1447 parts from src fslocal "/storage/snapshots/20221212194351-172EA3346AA94EFA"
2022-12-12T19:43:53.665Z	info	VictoriaMetrics/lib/backup/actions/backup.go:84	obtained 136 parts from dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.935Z	info	VictoriaMetrics/lib/backup/actions/backup.go:90	obtained 1494 parts from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.940Z	info	VictoriaMetrics/lib/backup/actions/backup.go:122	server-side copying 1136 parts from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.941Z	info	VictoriaMetrics/lib/backup/actions/backup.go:125	server-side copying part{path: "data/small/2022_09/535632568_4624033_20220930015512.441_20220930072925.800_17171BEA2F769ED7/values.bin", file_size: 114870012, offset: 0, size: 114870012} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.941Z	info	VictoriaMetrics/lib/backup/actions/backup.go:125	server-side copying part{path: "data/small/2022_09/19022_19022_20220915205131.856_20220930235958.277_17171BEA2F76CE7B/timestamps.bin", file_size: 0, offset: 0, size: 0} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.941Z	info	VictoriaMetrics/lib/backup/actions/backup.go:125	server-side copying part{path: "data/small/2022_09/19022_19022_20220915205131.856_20220930235958.277_17171BEA2F76CE7B/metaindex.bin", file_size: 731, offset: 0, size: 731} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.941Z	info	VictoriaMetrics/lib/backup/actions/backup.go:125	server-side copying part{path: "data/small/2022_09/535632568_4624033_20220930015512.441_20220930072925.800_17171BEA2F769ED7/timestamps.bin", file_size: 428092536, offset: 0, size: 428092536} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.941Z	info	VictoriaMetrics/lib/backup/actions/backup.go:125	server-side copying part{path: "data/small/2022_09/535632568_4624033_20220930015512.441_20220930072925.800_17171BEA2F769ED7/index.bin", file_size: 42583330, offset: 0, size: 42583330} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.941Z	info	VictoriaMetrics/lib/backup/actions/backup.go:125	server-side copying part{path: "data/small/2022_09/19022_19022_20220915205131.856_20220930235958.277_17171BEA2F76CE7B/index.bin", file_size: 125788, offset: 0, size: 125788} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.941Z	info	VictoriaMetrics/lib/backup/actions/backup.go:125	server-side copying part{path: "data/small/2022_09/19022_19022_20220915205131.856_20220930235958.277_17171BEA2F76CE7B/values.bin", file_size: 0, offset: 0, size: 0} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.942Z	info	VictoriaMetrics/lib/backup/actions/backup.go:125	server-side copying part{path: "data/small/2022_09/504495497_4621409_20220930105117.320_20220930134724.755_17171BEA2F76B109/timestamps.bin", file_size: 362707169, offset: 0, size: 362707169} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.942Z	info	VictoriaMetrics/lib/backup/actions/backup.go:125	server-side copying part{path: "data/small/2022_09/31_10_20220930235749.326_20220930235946.718_17171BEA2F76CE9B/metaindex.bin", file_size: 66, offset: 0, size: 66} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.942Z	info	VictoriaMetrics/lib/backup/actions/backup.go:125	server-side copying part{path: "data/small/2022_09/549208699_4614651_20220930173916.269_20220930214305.876_17171BEA2F76C7E1/timestamps.bin", file_size: 397423708, offset: 0, size: 397423708} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:53.986Z	info	VictoriaMetrics/lib/backup/actions/backup.go:125	server-side copying part{path: "data/small/2022_09/31_10_20220930235749.326_20220930235946.718_17171BEA2F76CE9B/timestamps.bin", file_size: 0, offset: 0, size: 0} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}
2022-12-12T19:43:59.142Z	info	VictoriaMetrics/lib/backup/actions/backup.go:133	server-side copied 8 out of 1136 parts from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} in 5.201486629s
2022-12-12T19:43:59.143Z	fatal	VictoriaMetrics/app/vmbackup/main.go:105	cannot create backup: cannot copy part{path: "data/small/2022_09/504495497_4621409_20220930105117.320_20220930134724.755_17171BEA2F76B109/timestamps.bin", file_size: 362707169, offset: 0, size: 362707169} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to dst AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}: cannot copy "data/small/2022_09/504495497_4621409_20220930105117.320_20220930134724.755_17171BEA2F76B109/timestamps.bin" from AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"} to AZBlob{container: "victoria-metrics", dir: "full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/"}: PUT https://mradminmetricsbackups.blob.core.windows.net/victoria-metrics/full/2022-12-12/victoria-metrics-victoria-metrics-cluster-vmstorage-1/data/small/2022_09/504495497_4621409_20220930105117.320_20220930134724.755_17171BEA2F76B109/timestamps.bin/00000000159E78E1_0000000000000000_00000000159E78E1
--------------------------------------------------------------------------------
RESPONSE 409: 409 The source request body for synchronous copy is too large and exceeds the maximum permissible limit (256MB).
ERROR CODE: CannotVerifyCopySource
--------------------------------------------------------------------------------
<?xml version="1.0" encoding="utf-8"?><Error><Code>CannotVerifyCopySource</Code><Message>The source request body for synchronous copy is too large and exceeds the maximum permissible limit (256MB).
RequestId:c6e898b5-e01e-000e-7f62-0e7906000000
Time:2022-12-12T19:43:53.9944153Z</Message></Error>
```

**Screenshots**
n/a

**Version**
I am using this image: `victoriametrics/vmbackup:v1.82.0`

**Used command-line flags**
Command can be seen above.

**Comment by zekker6** [comment]
Hi @mckennajones 
This issue have been fixed in https://github.com/VictoriaMetrics/VictoriaMetrics/pull/3479
Could you try running vmbackup with this image: `victoriametrics/vmbackup:heads-fix-backup-azremote-async-copy-0-gf65b0ccf8` and check if issue persists?

**Comment by valyala** [comment]
Re-opening the issue until the bugfix is integrated in the upcoming release.

**Comment by mckennajones** [comment]
@zekker6 thanks for the timely response! I'm happy to report that it works. Logs with repeated parts removed below for readability:

```
2022-12-13T17:28:51.453Z	info	VictoriaMetrics/lib/logger/flag.go:12	build version: vmbackup-20221213-122721-heads-fix-backup-azremote-async-copy-0-gf65b0ccf8
2022-12-13T17:28:51.453Z	info	VictoriaMetrics/lib/logger/flag.go:13	command-line flags
2022-12-13T17:28:51.453Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -dst="azblob://victoria-metrics/full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7"
2022-12-13T17:28:51.453Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -origin="azblob://victoria-metrics/latest/victoria-metrics-victoria-metrics-cluster-vmstorage-7"
2022-12-13T17:28:51.453Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -snapshot.createURL="http://victoria-metrics-victoria-metrics-cluster-vmstorage-7.victoria-metrics-victoria-metrics-cluster-vmstorage.victoria-metrics:8482/snapshot/create"
2022-12-13T17:28:51.453Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -storageDataPath="/storage"
2022-12-13T17:28:51.453Z	info	VictoriaMetrics/app/vmbackup/main.go:53	Snapshot create url http://victoria-metrics-victoria-metrics-cluster-vmstorage-7.victoria-metrics-victoria-metrics-cluster-vmstorage.victoria-metrics:8482/snapshot/create
2022-12-13T17:28:51.453Z	info	VictoriaMetrics/app/vmbackup/main.go:60	Snapshot delete url http://victoria-metrics-victoria-metrics-cluster-vmstorage-7.victoria-metrics-victoria-metrics-cluster-vmstorage.victoria-metrics:8482/snapshot/delete
2022-12-13T17:28:51.453Z	info	VictoriaMetrics/lib/snapshot/snapshot.go:28	Creating snapshot
2022-12-13T17:28:57.226Z	info	VictoriaMetrics/lib/snapshot/snapshot.go:52	Snapshot 20221213172856-173025FD13EC7316 created
2022-12-13T17:28:57.226Z	info	VictoriaMetrics/lib/httpserver/httpserver.go:96	starting http server at http://127.0.0.1:8420/
2022-12-13T17:28:57.226Z	info	VictoriaMetrics/lib/httpserver/httpserver.go:97	pprof handlers are exposed at http://127.0.0.1:8420/debug/pprof/
2022-12-13T17:28:57.279Z	info	VictoriaMetrics/lib/backup/actions/backup.go:72	starting backup from fslocal "/storage/snapshots/20221213172856-173025FD13EC7316" to AZBlob{container: "victoria-metrics", dir: "full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"} using origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"}
2022-12-13T17:28:57.285Z	info	VictoriaMetrics/lib/backup/actions/backup.go:78	obtained 747 parts from src fslocal "/storage/snapshots/20221213172856-173025FD13EC7316"
2022-12-13T17:28:57.288Z	info	VictoriaMetrics/lib/backup/actions/backup.go:84	obtained 0 parts from dst AZBlob{container: "victoria-metrics", dir: "full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"}
2022-12-13T17:28:57.476Z	info	VictoriaMetrics/lib/backup/actions/backup.go:90	obtained 716 parts from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"}
2022-12-13T17:28:57.478Z	info	VictoriaMetrics/lib/backup/actions/backup.go:122	server-side copying 592 parts from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"} to dst AZBlob{container: "victoria-metrics", dir: "full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"}
2022-12-13T17:28:57.479Z	info	VictoriaMetrics/lib/backup/actions/backup.go:125	server-side copying part{path: "data/small/2022_11/3855132_2822936_20221130235408.044_20221130235927.696_1726A34939E48154/timestamps.bin", file_size: 2083126, offset: 0, size: 2083126} from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"} to dst AZBlob{container: "victoria-metrics", dir: "full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"}

******* Server-side copy logs removed here *******

2022-12-13T17:28:59.909Z	info	VictoriaMetrics/lib/backup/actions/backup.go:133	server-side copied 592 out of 592 parts from origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"} to dst AZBlob{container: "victoria-metrics", dir: "full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"} in 2.430615404s
2022-12-13T17:28:59.910Z	info	VictoriaMetrics/lib/backup/actions/backup.go:143	uploading 155 parts from src fslocal "/storage/snapshots/20221213172856-173025FD13EC7316" to dst AZBlob{container: "victoria-metrics", dir: "full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"}
2022-12-13T17:28:59.910Z	info	VictoriaMetrics/lib/backup/actions/backup.go:146	uploading part{path: "data/small/2022_12/238245044_3273307_20221213133016.340_20221213143615.163_173025FDEDF78FEB/metaindex.bin", file_size: 92841, offset: 0, size: 92841} from src fslocal "/storage/snapshots/20221213172856-173025FD13EC7316" to dst AZBlob{container: "victoria-metrics", dir: "full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"}

******* Uploading new bytes logs removed here ******

2022-12-13T17:29:09.911Z	info	VictoriaMetrics/lib/backup/actions/backup.go:164	uploaded 2575857979 out of 3995921095 bytes from src fslocal "/storage/snapshots/20221213172856-173025FD13EC7316" to dst AZBlob{container: "victoria-metrics", dir: "full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"} in 10.000771403s
2022-12-13T17:29:19.910Z	info	VictoriaMetrics/lib/backup/actions/backup.go:164	uploaded 3609301949 out of 3995921095 bytes from src fslocal "/storage/snapshots/20221213172856-173025FD13EC7316" to dst AZBlob{container: "victoria-metrics", dir: "full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"} in 20.000423525s
2022-12-13T17:29:29.910Z	info	VictoriaMetrics/lib/backup/actions/backup.go:164	uploaded 3883762711 out of 3995921095 bytes from src fslocal "/storage/snapshots/20221213172856-173025FD13EC7316" to dst AZBlob{container: "victoria-metrics", dir: "full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"} in 30.000229469s
2022-12-13T17:29:34.508Z	info	VictoriaMetrics/lib/backup/actions/backup.go:164	uploaded 3995921095 out of 3995921095 bytes from src fslocal "/storage/snapshots/20221213172856-173025FD13EC7316" to dst AZBlob{container: "victoria-metrics", dir: "full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"} in 34.597686817s
2022-12-13T17:29:34.508Z	info	VictoriaMetrics/lib/backup/actions/backup.go:171	backup from src fslocal "/storage/snapshots/20221213172856-173025FD13EC7316" to dst AZBlob{container: "victoria-metrics", dir: "full/test/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"} with origin AZBlob{container: "victoria-metrics", dir: "latest/victoria-metrics-victoria-metrics-cluster-vmstorage-7/"} is complete; backed up 219375854090 bytes in 37.229 seconds; deleted 0 bytes; server-side copied 215379932995 bytes; uploaded 3995921095 bytes
2022-12-13T17:29:34.516Z	info	VictoriaMetrics/app/vmbackup/main.go:112	gracefully shutting down http server for metrics at ":8420"
2022-12-13T17:29:34.516Z	info	VictoriaMetrics/app/vmbackup/main.go:116	successfully shut down http server for metrics in 0.000 seconds
2022-12-13T17:29:34.516Z	info	VictoriaMetrics/lib/snapshot/snapshot.go:63	Deleting snapshot 20221213172856-173025FD13EC7316
2022-12-13T17:29:34.535Z	info	VictoriaMetrics/lib/snapshot/snapshot.go:90	Snapshot 20221213172856-173025FD13EC7316 deleted

```

**Comment by valyala** [comment]
The bugfix has been included in [vmbackup v1.85.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed then.
